### ISSUE TITLE
Machinist table - Not Resolved - 7.6.3

### ISSUE THREAD

**Issue by TheKye** [StrangeLoopGames/EcoIssues#8848]
There seems to be some issues regarding Craft-able items in this table,

Lathe - Requires Screw Press, screw press doesn't act as a module

Shaper - Requires Lathe - Can't Build Lathe As Stated Above

Boiler - Requires Shaper - Can't Build Shaper As Stated Above

The only possible/ current fix is removing the requirements from the Recipes, but that defeats the purpose of building them in some aspect

This is in relation to #8779 that was closed but the error is still not fixed, i have edited the code in the files themselves to make these items usable and the recipes for the items that they make, 

We have tried building a new room ad placing the modules down using admin commands and the error is still the same. please Re-Check this issue using Vanilla files from 7.6.3 to confirm the error.

**Comment by elfl0rd** [comment]
All work as intended, you need screw press and another benches in same room.
@D3nnis3n can you close? :)

**Comment by D3nnis3n** [comment]
Thats correct, but it's not made aware to players that those machines need to be in the same room and it should be, keeping open therefore.

**Comment by elfl0rd** [comment]
But "missing info" is different issue from this :)

**Comment by Deantwo** [comment]
Ooh this is still not fixed?
Nor really a duplicate I guess, but fir old context: #9040
